MAUI/TreeMap/migration.md

```diff
@@ -0,0 +1,1450 @@
+
+using Syncfusion.Maui.TreeMap;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfTreeMap treeMap = new SfTreeMap();
+        PopulationViewModel viewModel = new PopulationViewModel();
+        treeMap.DataSource = viewModel.PopulationDetails;
+        this.treeMap.LeafItemBrushSettings = new TreeMapDesaturationBrushSettings() { Brush = Brush.BlueViolet, From = 1, To = 0.2 };
+        this.Content = treeMap;
+    }
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="ViewModel.cs" %}
+
+public class PopulationViewModel
+{
+    public PopulationViewModel()
+    {
+        this.PopulationDetails = new ObservableCollection<PopulationDetails>()
+            {
+                new PopulationDetails() { Continent ="North America", Country = "United States of America", Population = 339996564 },
+                new PopulationDetails() { Continent ="North America", Country = "Mexico", Population = 128455567 },
+                new PopulationDetails() { Continent ="North America", Country = "Canada", Population = 38781292 },
+                new PopulationDetails() { Continent ="South America", Country = "Bolivia", Population = 12388571 },
+                new PopulationDetails() { Continent ="North America", Country = "Costa Rica", Population = 5212173 },
+                new PopulationDetails() { Continent ="South America", Country = "Uruguay", Population = 3423109 },
+            };
+    }
+    public ObservableCollection<PopulationDetails> PopulationDetails
+    {
+        get;
+        set;
+    }
+}
+
+{% endhighlight %}
+{% highlight c# tabtitle="Model.cs" %}
+
+public class PopulationDetails
+{
+    public string Country { get; set; }
+    public string Continent { get; set; }
+    public int Population { get; set; }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<table>
+<tr>
+<th>Xamarin SfTreeMap control</th>
+<th>.NET MAUI SfTreeMap control</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[Color](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.DesaturationColorMapping.html#Syncfusion_SfTreeMap_XForms_DesaturationColorMapping_Color)'| markdownify }}</td>
+<td>Brush</td>
+<td>Gets or sets the color value for the tree map item. This is used to set color for each tree map item.</td>
+</tr>
+
+<tr>
+<td>{{'[From](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.DesaturationColorMapping.html#Syncfusion_SfTreeMap_XForms_DesaturationColorMapping_From)'| markdownify }}</td>
+<td>From</td>
+<td>Gets or sets the from value for desaturation. The From value is used to point where the desaturation starts.</td>
+</tr>
+
+<tr>
+<td>{{'[To](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.DesaturationColorMapping.html#Syncfusion_SfTreeMap_XForms_DesaturationColorMapping_To)'| markdownify }}</td>
+<td>To</td>
+<td>Gets or sets the two values for desaturation. The To value is used to point where the desaturation ends.</td>
+</tr>
```

Comment:
Here,  do we need to mention all XForms properties, which we removed in maui based on use case?